corehq/ex-submodules/couchforms/getters.py

```diff
@@ -69,13 +72,32 @@ def get_instance_and_attachment(request):
     return instance, attachments
 
 
-def _valid_file_extension(file):
+def _valid_instance_file_extension(file):
     if "." not in file.name:
         return False
     file_extension = file.name.rsplit(".", 1)[-1]
     return file_extension == 'xml'
 
 
+def _valid_attachment_file(file):
+    return _valid_attachment_file_extension(file) or _valid_attachment_file_mimetype(file)
+
+
+def _valid_attachment_file_extension(file):
+    if "." not in file.name:
+        return False
+    file_extension = file.name.rsplit(".", 1)[-1]
+    return file_extension in VALID_ATTACHMENT_FILE_EXTENSIONS
+
+
+def _valid_attachment_file_mimetype(file):
+    return (
+        file.content_type.startswith(("audio/", "image/", "video/"))
+        # default mimetype set by CommCare
+        or file.content_type == "application/octet-stream"
```

Comment:
Formplayer also allows `application/pdf`

Reply:
Ah, bad miss.
Thanks for catching that @snopoke